index.bs

```diff
@@ -11144,23 +11144,25 @@ in which case they are exposed on every object that [=implements=] the interface
 
     1.  Let |steps| be the following series of steps:
         1.  Try running the following steps:
-            1.  Let |O| be <emu-val>null</emu-val>.
+            1.  Let |object| be null.
             1.  If |target| is an [=interface=], and |attribute| is a [=regular attribute=]:
                 1.  If the <emu-val>this</emu-val> value is <emu-val>null</emu-val> or
-                    <emu-val>undefined</emu-val>, set |O| to |realm|'s [=Realm/global object=].
+                    <emu-val>undefined</emu-val>, let |O| be |realm|'s [=Realm/global object=].
```

Comment:
We don’t use the ECMA-262-like variable scoping scheme here, but the [Infra](https://infra.spec.whatwg.org/#variables)-based block-scoping rules.

Reply:
Can you show what this would look like? Ideally we'd have Rust-style `let x = if .. {} else {}`.

Reply:
Basically you'd declare the _O_ upfront and then override it here.

Reply:
Rust style is done via "Let x be y if z, or w otherwise."